File: CS/CS.NET-VB.NET/ASP.NET 4.0 Book/Source/Chapter 29/CS/ComplexdataAppCS/App_Code/Classprofile.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.Profile;

/// <summary>
/// Summary description for Classprofile
/// </summary>
public class Classprofile
{
	
		private string Dy;
	private string Mnth;
	private string Yr;
	public string Day
	{
		get { return Dy; }
		set { Dy = value; }
	}
	public string Month
	{
		get { return Mnth; }
		set
		{
			Mnth = value;
		}
	}
	public string Year
	{
		get { return Yr; }
		set
		{
			Yr = value;
		}
	}

	
}

File: CS/CS.NET-VB.NET/ASP.NET 4.0 Book/Source/Chapter 29/VB/ComplexdataAppVB/App_Code/Classprofile.vb
Imports Microsoft.VisualBasic
Imports System.Web.Profile
Public Class Classprofile
    Private Dy As String
    Private Mnth As String
    Private Yr As String
    Public Property Day() As String
        Get
            Return Dy
        End Get
        Set(ByVal value As String)
            Dy = value
        End Set
    End Property
    Public Property Month() As String
        Get
            Return Mnth
        End Get

        Set(ByVal value As String)
            Mnth = value
        End Set
    End Property

    Public Property Year() As String
        Get
            Return Yr
        End Get

        Set(ByVal value As String)
            Yr = value
        End Set
    End Property

End Class
